PointOfDispensingApp/src/brokerRequests.js

```diff
@@ -35,30 +35,8 @@ export default {
 
   // Get patient from QR code for patient retrieval
   getPatientFromQrCode: (qrCode) => {
-    
-    // If we didn't get a Qr Code, return David's patient data
-    if(!qrCode) {
```

Comment:
Maybe we should still have this check? If sample data was returned during the functional exercise, that means there were issues scanning the QR